Makefile

```diff
@@ -159,12 +159,14 @@ define publish_image
 	${1} push ${3}
 endef # publish_image
 
-publish:
+publish_latest:
 	$(call publish_image,docker,${SERVICE},quay.io/ocpmetal/assisted-service:latest)
-	$(call publish_image,docker,${SERVICE},quay.io/ocpmetal/assisted-service:${GIT_REVISION})
 	$(call publish_image,docker,${ISO_CREATION},quay.io/ocpmetal/assisted-iso-create:latest)
-	$(call publish_image,docker,${ISO_CREATION},quay.io/ocpmetal/assisted-iso-create:${GIT_REVISION})
 	$(call publish_image,podman,${SERVICE_ONPREM},quay.io/ocpmetal/assisted-service-onprem:latest)
+
+publish:
```

Comment:
Please change 3rd parameter to be `TAG` or something so we can use the same target twice
Something like
```
publish PUBLISH_TAG=latest
publish PUBLISH_TAG=${GIT_REVISION}
```

Reply:
not sure what parameter you mean

Reply:
Instead of having two Makefile targets doing the same thing except for one parameter, let's merge them into one target that receives a parameter.